cursorless-talon-dev/src/cursorless_launch_configuration.talon-list

```diff
@@ -0,0 +1,19 @@
+app: vscode
+win.title: /cursorless/
+list: user.cursorless_launch_configuration
```

Comment:
I'm pretty sure `list:` needs to be on the first line. Have you tested this?

Reply:
done